tests/VirtoCommerce.CatalogModule.Tests/ProductAssociationTests.cs

```diff
@@ -103,101 +104,6 @@ public async Task UpdateAssociationsAsync_AddNewAssociation_Added(dataModel.Asso
             _catalogRepositoryMock.Verify(x => x.Add(It.Is<dataModel.AssociationEntity>(q => q.AssociatedItemId == "NEW_AssociatedItemId")), Times.Once);
         }
 
-
```

Comment:
The SaveChanges can't be tested simple. Because we do not use entityFramework add or remove command. Also we make a copy of entities that existed in database and can't access to it outside of the method.